application.go

```diff
@@ -272,11 +351,45 @@ func (r *Application) CheckTCP(port, interval int) (*Application, error) {
 	health.IntervalSeconds = interval
 	health.PortIndex = portIndex
 	// step: add to the checks
-	r.HealthChecks = append(r.HealthChecks, health)
+	r.AddHealthCheck(*health)
 
 	return r, nil
 }
 
+// AddUris adds one or more uris to the applications
+//		arguments:	the uri(s) you are adding
+func (r *Application) AddUris(newUris ...string) *Application {
```

Comment:
Initialisms in Go are normally all-caps ([reference](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)), so I'd name it `[aA]ddURIs` in comments, methods, and variables. I know that's not consistent with other parts of go-marathon but believe we should start trying to be more in line with existing Go conventions.